neard/src/cli.rs

```diff
@@ -670,6 +682,52 @@ pub enum VerifyProofError {
     InvalidBlockHashProof,
 }
 
+#[derive(clap::Parser)]
+pub struct VerifyTrieProofSubCommand {
```

Comment:
mention in the comment, that this verifies that a given key belongs to a given account.